app/controllers/weets.js

```diff
@@ -0,0 +1,6 @@
+const weetService = require('../services/weets');
+
+exports.getQuote = async (_, res) => {
+    const data = await weetService.getRandomQuote();
```

Comment:
Remember to use the informational logs, this will make it much easier to debug the API.